tenpy/simulations/time_evolution.py

```diff
@@ -74,3 +82,411 @@ def final_measurements(self):
         We already performed a set of measurements after the evolution in :meth:`run_algorithm`.
         """
         pass
+
+
+class TimeDependentCorrelation(RealTimeEvolution):
+    r"""A subclass of :class:`RealTimeEvolution` to specifically calculate the
```

Comment:
single line. Maybe:
```
Specialized :class:`RealTimeEvolution` to calculate a time dependent correlation function of a ground state.
```
Mention that psi_0 needs to be an eigenstate of H for this variant in the doc string below, and refer to the `TimeDependentCorrelationEvolveBraKet` as generalization for arbitrary initial states